### DISCUSSION TITLE
Assistance Needed for Digit Setup on SDC Server

### DISCUSSION THREAD

**Discussion by PrasadAthani** [egovernments/Digit-Core#532] [Local Governance]
Dear Team,

We are currently in the process of setting up Digit for OBPAS services on the SDC server. While we have successfully containerised most of the backbone and core services, we are encountering challenges with a few core, business, and municipal services due to their interdependencies.

We are following the deployment flow as outlined below:

 **Core Services > Business Services > Municipal Services**

The specific core services where we are facing issues are as follows:

1. egov-indexer
2. egov-persister
3. egov-pg-service
4. egov-searcher
5. egov-url-shortening
6. egov-user
 
All other core and backbone services are up and running smoothly.

We would greatly appreciate your guidance in resolving these challenges so we can complete the setup.

Thank you in advance for your support.

**Comment by naresh-egov** [comment]
@PrasadAthani Please share the pod logs and events, so we can better understand the issue and work towards resolving it.
